test/sql/sample/table_samples/test_sample_is_destroyed_on_updates.test

```diff
@@ -0,0 +1,122 @@
+# name: test/sql/sample/table_samples/test_sample_is_destroyed_on_updates.test
+# description: Test sampling of larger relations
+# group: [table_samples]
+
+require vector_size 2048
```

Comment:
Maybe we should set the sample size separately from the vector size so that these tests can run with vsize = 2